### PR TITLE
Registry new endpoint for removing service manually from XSubscription List

### PR DESCRIPTION
**Pull request by him3010**
Removing service only from the Xsubscription List based on service name and version but later can be added to remove it from registered instances list (leaving that functionality same).

### CODE REVIEW

File: vyked/registry.py [hunk 1]
```diff
@@ -499,6 +504,11 @@ def _show_current_state(self, protocol):
                        'XSubscription List': self._repository._subscribe_list,
                        'Service Dependencies': self._repository._service_dependencies}
         protocol.send(status_dict)
+        
+    def _remove_service_from_xsubscribe(self, packet, protocol):
+        params = packet['params']
+        self._repository._remove_service_from_xsubscribe(params['service'], params['version'])
```
Comment (swapniljain14) [inline vyked/registry.py]: .lower() should be used for service I guess

File: vyked/registry.py [hunk 2]
```diff
@@ -188,6 +182,15 @@ def _get_full_instance_name(service: str, version, node_id):
     def _split_key(key: str):
         return tuple(key.split('/'))
 
+    def _remove_service_from_xsubscribe(self, service, version):
```
Comment (swapniljain14) [inline vyked/registry.py]: Since this is not private, remove the starting '_' from method name

**Review by swapniljain14 — CHANGES_REQUESTED** [review]
Looks fine, commented in-line for 2 small changes

File: vyked/registry.py (hunk 1 above)
Comment (him3010) [reply vyked/registry.py]: we can use but maintaining consistency as lower is not used in many packets type, 
though we can change that

**Review by swapniljain14 — APPROVED** [review]

**Pull request merged**